src/web-exploitation/intro-to-sqli/blind-injections.md

```diff
@@ -56,32 +78,33 @@ Hint: First use auth by-pass then start applying blind SQL.
 This is based on a real CTF question, from the RaziCTF 2020.
 In this question you are given a login page and a `/login` endpoint we can post credentials to. When trying to log in it just says `You are not allowed to login` which seems quite odd. 
 ```
-http://130.185.122.155:8080/login 
+http://example.com/login 
```

Comment:
Doesn't make sense to explain a challenge if the user can't access it.
From the user's POV, this section is supposed to be practical and not another reading exercise.